framework/db/Query.php

```diff
@@ -844,6 +849,119 @@ public function addParams($params)
     }
 
     /**
+     * @var array aliases defined for tables used in this query. `tablename => alias`.
```

Comment:
> `tablename => alias`

I am unsure: does this means single table name can have only one alias?
Was not original issue about using same table twice inside single query?

For example, I may need to compose following SQL:
```
SELECT u.*, COUNT(sent_m) AS sentCount, COUNT(received_m) AS receiveCount
FROM User AS u
LEFT JOIN PrivateMessage AS sent_m ON sent_m.senderId = u.id
LEFT JOIN PrivateMessage AS received_m ON received_m.receiverId = u.id
GROUP BY u.id
```

What should I do then?